.github/workflows/release_github.yaml

```diff
@@ -0,0 +1,19 @@
+# On each new commit to master, create/update a PR with release
+# automatically bumps version and creates changelog as per conventional commits
+name: Release Github
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: GoogleCloudPlatform/release-please-action@v2
+        id: release
+        with:
+          token: ${{ secrets.REPO_GHA_PAT }}
```

Comment:
@vandot Do we need to use PAT here? (So that it triggers the `new tag` event and therefore the publish to GH Pages workflow https://github.com/ethersphere/bee-js.github.io/blob/master/.github/workflows/gh-pages.yaml )

If so, can you add it please?

Reply:
@vojtechsimetka really not sure, added it